recipes/libignition-rendering/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake .. \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_PREFIX_PATH=$PREFIX -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=True \
+      -DCMAKE_C_FLAGS=-DGLX_GLXEXT_LEGACY \
```

Comment:
I am unsure if this still passes through the CFLAGS and CXXFLAGS from the environment.

I think it could be better to change this for 

```
export CXXFLAGS="${CXXFLAGS} -DGLX_GLXEXT_LEGACY
....
```

before invoking cmake to be sure to pass the CXXFLAGS from the conda-forge compilers to cmake.

Reply:
Good point, updated here 5df9a9f8c76ad4180ae16bb13871f451eab094a2